spoti-vote-backend/src/ServerInstance.js

```diff
@@ -78,37 +102,71 @@ method.fetchData = async function() {
     console.log('User ' + this.host.id + ' logged in all data was fetched.');
 }
 
+/**
+* Returns the host data
+*
+* @author: Michiocre
+* @return {object} The host object
```

Comment:
is somewhere documented what a host object contains?

Reply:
Yes in the constructor of the Room